src/main/java/seedu/address/logic/commands/editcommand/EditParticipantCommand.java

```diff
@@ -55,11 +55,12 @@ public CommandResult execute(Model model) throws CommandException {
          *     throw new CommandException(MESSAGE_DUPLICATE_PERSON);
          * }
          */
-        if (model.updateParticipant(this.id, editedParticipant)) {
-            // model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        try {
+            model.updateParticipant(this.id, editedParticipant);
             return new CommandResult(String.format(MESSAGE_EDIT_PARTICIPANT_SUCCESS, editedParticipant.toString()));
+        } catch (AlfredException e) {
+            return new CommandResult(MESSAGE_DUPLICATE_PARTICIPANT);
```

Comment:
Think Joanna said throw exception. And how are we differentiating between no team and duplicate participant again?

Reply:
Ah yea actually not too sure about this, just modified so the project builds